.circleci/config.yml

```diff
@@ -29,18 +30,19 @@ references:
   restore-dependencies-cache: &restore-dependencies-cache
     restore_cache:
       keys:
-        - deps-2-py<< parameters.version >>-{{ checksum ".checksum" }}
+        - deps-3-py<< parameters.version >>-{{ checksum ".checksum" }}
   install-dependencies: &install-dependencies
     run:
       name: Install Dependencies
       command: |
         python3 -m venv venv
         venv/bin/pip install -U pip setuptools wheel
-        venv/bin/pip install -U 'Sphinx <7'
+        # Install docs separately, first, due to issues with Sphinx
+        venv/bin/pip install -r script/docs/requirements.txt
```

Comment:
I'm pretty sure we can eventually do away with this, however it needs a deeper reworking of how we do builds & docs builds, so for now I'm sticking with improving the workaround we already have.